doc/api/pyplot_api.rst

```diff
@@ -0,0 +1,283 @@
+******
+pyplot
+******
+
+.. currentmodule:: matplotlib.pyplot
+
+.. automodule:: matplotlib.pyplot
+    :no-members:
+
+Information
+===========
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    colormaps
+    get_plot_commands
+
+Global functions
+================
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    findobj
+    install_repl_displayhook
+    ioff
+    ion
+    isinteractive
+    pause
+    rc
+    rc_context
+    rcdefaults
+    show
+    switch_backend
+    uninstall_repl_displayhook
+
+Any artist
+==========
+
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
+
+    getp
```

Comment:
`getp` does not link because it does not live in pyplot. It's just imported. Can we do something about this?